ports/servoshell/minibrowser.rs

```diff
@@ -107,10 +108,20 @@ impl Minibrowser {
                 self.last_mouse_position
                     .map_or(false, |p| self.is_in_browser_rect(p))
             },
-            winit::event::WindowEvent::MouseWheel { .. } |
-            winit::event::WindowEvent::MouseInput { .. } => self
-                .last_mouse_position
-                .map_or(false, |p| self.is_in_browser_rect(p)),
```

Comment:
It looks like this change no longer checks to see if the last mouse position was in the browser rectangle? What's going on here?

Reply:
Yeah, I thought I made a way around i.e I simplified things. But it isn't the case. I will try to do some changes.